### PR TITLE
Fix bazel

### PR DESCRIPTION
**Pull request by liztio**
**What this PR does / why we need it**:

`bazel test //cmd/kubeadm/...` has been broken for a while. This is incredibly annoying to me, because I use it constantly during my workflow. This should fix it.

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 

thanks for the fix!

**Review by fabriziopandini — COMMENTED** [review]
@liztio Thanks for the fix!
/approve
/lgtm

File: cmd/kubeadm/test/cmd/token_test.go [hunk 1]
```diff
@@ -28,15 +29,26 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\.\\S{16}\n$"
 )
 
-var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
+var kubeadmPathFlag = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```
Comment (ash2k) [inline cmd/kubeadm/test/cmd/token_test.go]: `filepath.Join(os.Getenv("KUBE_ROOT"), "cluster", "kubeadm.sh")` to use platform-specific separator.

File: cmd/kubeadm/test/cmd/token_test.go [hunk 2]
```diff
@@ -28,15 +29,26 @@ const (
...
+
+func getKubeadmPath() string {
+	kubeadmPath := *kubeadmPathFlag // TEST_SRCDIR is provided by Bazel.
+	if srcDir := os.Getenv("TEST_SRCDIR"); srcDir != "" {
+		kubeadmPath = path.Join(srcDir, os.Getenv("TEST_WORKSPACE"), kubeadmPath)
```
Comment (ash2k) [inline cmd/kubeadm/test/cmd/token_test.go]: `path.Join(...)` -> `filepath.Join(...)`

**Pull request merged**